MEMBER_EXPECTATIONS.md

```diff
@@ -0,0 +1,57 @@
+members of leadership teams including the
+([CPC](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md).
+
+It is understood that members will have individual opinions and will express
+their individual opinions during discussions within the organization. All
+contributors are expected to be respectful of these different viewpoints,
+and to work collaboratively with one another in a manner that constructively
+elevates all contributors.
+
+When decisions are made within the established guidelines and policies of the
+OpenJS Foundation projects or groups, those in leadership roles have a
+responsibility to uphold and respect the decision even if they disagree with
+it. This is especially important in external communications, for example in
+social media. Should the member be unwilling or unable to do so, then they
+should resign their leadership position. This does not mean that decisions
+cannot be revisited and discussed within the team at a later time.
+
+Members of our leadership groups must also conduct themselves in a
+professional and respectful manner. Some general guidelines include:
+
+- Remediate quickly when you realize you made a mistake. Leaders are human,
+  and they will make mistakes. However they should act swiftly to
+  acknowledge mistakes and correct them.
+- Aim to remediate first and then discuss. If other members of the
+  team express concerns about actions, acknowledge their concerns by
+  stopping the actions in question and then discuss within the team
+  to come to a common agreement.
+- Treat all community members with respect, consideration, and highest
+  standards of ethical conduct.
+- Value a diversity of views and opinions. Avoid preferential
+  treatment, and hold everyone (including ourselves) accountable to the same
+  set of standards. Everyone gets to speak up.
+- Deal with issues directly with the person in question. Resist complaining
+  about others in the OpenJS Foundation projects or team in a public sphere.
+- Build trust by keeping your promises.
+- Be the model of accountability and leadership. Provide the example of
+  ownership and stewardship that everyone can follow to success.
+- Commit to ongoing development and learning best practices for governing.
+- Criticize ideas rather than people, discussing any concerns in person
+  whenever possible, and taking responsibility for our statements.
+
+While the guidelines above focus primarily on the spaces where
+we participate in official OpenJS foundation work (GitHub, IRC, meetings,
+conferences), it is important to recognize that the public behavior
+of members also reflects on the OpenJS Foundation projects and groups.
+
+If you're interested in an introduction to diversity, inclusion, and unconscious bias, 
+[try this free training offered by our partners at the Linux Foundation](https://training.linuxfoundation.org/linux-courses/open-source-compliance-courses/inclusive-speaker-orientation).
```

Comment:
Seems to be a redirect to https://training.linuxfoundation.org/training/inclusive-speaker-orientation/.